src/modules/feedback/components/FeedbackModal.js

```diff
@@ -57,34 +57,44 @@ type PropsType = {|
   theme: ThemeType
 |}
 
+type SendingStatusType = 'IDLE' | 'SUCCESS' | 'ERROR'
+
 type StateType = {|
-  feedbackSent: boolean
+  feedbackSent: SendingStatusType
 |}
 
 export class FeedbackModal extends React.Component<PropsType, StateType> {
-  state = { feedbackSent: false }
+  state = { feedbackSent: 'IDLE' }
 
-  handleSubmit = () => this.setState({ feedbackSent: true })
+  handleSubmit = (value: SendingStatusType) => {
+    this.setState({ feedbackSent: value })
+  }
 
   handleOverlayClick = () => {
-    this.setState({ feedbackSent: false })
+    this.setState({ feedbackSent: 'IDLE' })
     this.props.closeFeedbackModal()
   }
 
   render () {
     const { theme, feedbackStatus, ...otherProps } = this.props
     const { feedbackSent } = this.state
 
+    let boxContainer
```

Comment:
We want to avoid `let` because it may be reassigned later on. Either use a separate `render(Feedback)Content` method returning the right content (`FeedbackBoxContainer` or `FeedbackThanksMessage` or use the following:
```suggestion
    const boxContainer = ['IDLE', 'ERROR'].includes(feedbackSent) ? 'FeedbackBoxContainer` : `FeedbackThanksMessage`
```
Of course you would have to use the actual components instead of my simplified version.

Reply:
Refactored with `renderBox()`